docs/serving/samples/telemetry-go/Dockerfile

```diff
@@ -14,14 +14,14 @@
 
 FROM golang AS builder
 
-WORKDIR /go/src/github.com/knative/docs/
+WORKDIR /go/src/github.com/knative/docs/docs/
 ADD . /go/src/github.com/knative/docs/
```

Comment:
Not in scope of this PR, but the way the builder section is set up now has a number of things that seem odd.

* Copying more than the sample into the container
* The current `ADD` instead of `COPY . .`
* Running build from a higher-level directory path

Also, this would be even more different in switching to use go modules.

Reply:
I guess the sample try to use the third part package in the `vendor` for compiler.
Fo the `ADD` and the other thing, I think maybe it's individual preference 😄